lib/livebook/config.ex

```diff
@@ -430,4 +437,8 @@ defmodule Livebook.Config do
     IO.puts("\nERROR!!! [Livebook] " <> message)
     System.halt(1)
   end
+
+  def allowed_uri_schemes!(env) do
+    System.get_env(env)
+  end
```

Comment:
When reading the env we can parse it, so that we always store a list in the config :)

```suggestion

  @doc """
  Parses and validates allowed URI schemes from env.
  """
  def allowed_uri_schemes!(env) do
    if schemes = System.get_env(env) do
      String.split(schemes, ",", trim: true)
    end
  end
```